content/en/docs/instrumentation/java/manual.md

```diff
@@ -55,93 +271,277 @@ telemetry source-identifying info.
+        </dependency>
+        <dependency>
+          <groupId>io.opentelemetry</groupId>
+          <artifactId>opentelemetry-sdk-extension-autoconfigure-spi</artifactId>
+        </dependency>
+    </dependencies>
+</project>
+```
+
+{{% /tab %}} {{< /tabpane>}}
+
+It allows you to auto-configure the OpenTelemetry SDK based on a standard set of
+supported environment variables and system properties. Each environment variable
+has a corresponding system property named the same way but as lower case and
+using the `.` (dot) character instead of the `_` (underscore) as separator.
+
+The logical service name can be specified via the `OTEL_SERVICE_NAME`
+environment variable (or `otel.service.name` system property).
+
+The traces, metrics or logs exporters can be set via the `OTEL_TRACES_EXPORTER`,
+`OTEL_METRICS_EXPORTER` and `OTEL_LOGS_EXPORTER` environment variables. For
+example `OTEL_TRACES_EXPORTER=logging` configures your application to use an
+exporter that writes all traces to the console. The corresponding exporter
+library has to be provided in the classpath of the application as well.
+
+For debugging and local development purposes, use the `logging` exporter. After
+you have finished setting up manual instrumentation, provide an appropriate
+exporter library in the classpath of the application to
+[export the app's telemetry data](/docs/instrumentation/java/exporters/) to one
+or more telemetry backends.
+
+The SDK auto-configuration has to be initialized from your code in order to
+allow the module to go through the provided environment variables (or system
+properties) and set up the `OpenTelemetry` instance by using the builders
+internally.
+
+```java
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
+
+OpenTelemetrySdk sdk = AutoConfiguredOpenTelemetrySdk.initialize()
+    .getOpenTelemetrySdk();
+```
 
-### Imports
+In the case of the [example app](#example-app) the the `DiceApplication` class
+gets updated as follows:
 
 ```java
+package otel;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.Banner;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
+
+@SpringBootApplication
+public class DiceApplication {
+  public static void main(String[] args) {
+    SpringApplication app = new SpringApplication(DiceApplication.class);
+    app.setBannerMode(Banner.Mode.OFF);
+    app.run(args);
+  }
+
+  @Bean
+  public OpenTelemetry openTelemetry() {
+    return AutoConfiguredOpenTelemetrySdk.initialize().getOpenTelemetrySdk();
+  }
+}
+```
+
+To verify your code, build and run the app:
+
+```sh
+gradle assemble
+env \
+OTEL_SERVICE_NAME=dice-server \
+OTEL_TRACES_EXPORTER=logging \
+OTEL_METRICS_EXPORTER=logging \
+OTEL_LOGS_EXPORTER=logging \
```

Comment:
Nothing in this setup is bridging logs into opentelemetry. Maybe consider using one of the appenders as described in [log appenders](https://opentelemetry.io/docs/instrumentation/java/manual/#log-appenders)?

Reply:
At this stage of the documentation no telemetry will be emitted, later I will update the log appenders section (and metrics) to also mention that demo application, I just started with tracing for now